schemas/semconv.schema.json

```diff
@@ -444,6 +457,13 @@
 				},
 				"type": {
 					"$ref": "#/$defs/AttributeType"
+				},
+				"stability": {
+					"allOf": [
+						{
+							"$ref": "#/$defs/StabilityLevel"
+						}
+					]
```

Comment:
I noticed that `AttributeFullSpec` had `stability` as required but without the property!